companies/515820587.json

```diff
@@ -1,26 +1,26 @@
 {
   "name": "Daisy",
-  "description": "Property Management Solutions",
+  "description": "Tech-driven, full service condo and co-op management company",
   "categoryId": 10,
   "size": "m",
-  "logoFile": "logos/daisy_investments-logo.webp",
+  "logoFile": "https://i.ibb.co/HCmwr6c/Daisy-Logo-Color-3x.png",
   "websiteUrl": "https://www.joindaisy.com/",
-  "careersUrl": null,
-  "isHiring": false,
-  "linkedinId": null,
+  "careersUrl": "https://www.joindaisy.com/careers",
+  "isHiring": true,
+  "linkedinId": "joindaisy",
   "crunchbaseId": null,
   "finderId": "daisy",
   "xId": null,
   "addresses": [
     {
-      "street": "וינהויז זאב",
-      "houseNumber": 3,
-      "city": "יהוד-מונוסון",
-      "lat": 32.0294773,
-      "lon": 34.8885912
+      "street": "בן אביגדור",
+      "houseNumber": 19,
+      "city": "תל אביב - יפו",
+      "lat": 32.067030,
```

Comment:
I'm not sure how the map extracts lat long from the address but here are the correct values from Google Maps:
lat: 32.066983, long: 34.790784